pyprika/recipe.py

```diff
@@ -50,7 +52,9 @@ class Recipe(object):
     notes = ''
     ingredients = ()
     directions = ()
+    categories = []
```

Comment:
I think this should be a tuple too... matter of fact it would be better just to delete `ingredients` and `directions` here and leave them to be set by `__init__`.